teams/wg-rust-by-example.toml

```diff
@@ -1,5 +1,6 @@
-name = "rust-by-example"
+name = "wg-rust-by-example"
 subteam-of = "docs"
```

Comment:
I left it as a subteam of docs because I think otherwise there's no way to distinguish between full teams and WGs?

Reply:
See https://github.com/rust-lang/team/pull/298#issuecomment-599582318.

Reply:
Looks like that's not allowed actually because Docs is archived...

Reply:
There's been some talk of a "learning" working group. Perhaps at some point this can be a subteam of that if it actually turns into something.

Reply:
Perhaps removing this for now is best. This can be addressed in a later PR.